### PR TITLE
Combo box icon

### PR DESCRIPTION
**Pull request by jmenghan**
## Related Issue
Please use one of the well-known [github fixes keywords](https://help.github.com/en/articles/closing-issues-using-keywords) to reference
the issues fixed with this PR (eg Fixes #<github issue number>). If an issue doesn't yet exist please create one if reasonable to aid 
in issue tracking.

**NOTE**: For multiple issues resolved by this PR use the corresponding keywords **every time** in a comma-delimited list per the reference
page above.

## Description
Added drop down icon for ComboBox from momentum.design

## Sample Card
Light
Drop_Light

Dark
Drop_Dark

## How Verified
How you verified the fix, including one or all of the following: 
1. New unit tests that were added if any. If none were added please add a quick line explaining why not.
2. Existing relevant unit/regression tests that you ran
3. Manual scenario verification if any; ***Do include .gif's or screenshots of the testing you performed here if you think that it 
will aid in code reviews or corresponding fixes on other platforms for eg.***

### CODE REVIEW

File: source/qml/Library/RendererQml/AdaptiveCardQmlRenderer.cpp [hunk 1]
```diff
@@ -535,7 +555,9 @@ namespace RendererQml
         backgroundTag->Property("color", choiceset.backgroundColor);
         backgroundTag->Property("border.color", "'grey'");
         backgroundTag->Property("border.width", "1");
+		backgroundTag->Property("implicitHeight", "35"); //TODO: To be replaced from hostconfig value
```
Comment (rj100189) [inline source/qml/Library/RendererQml/AdaptiveCardQmlRenderer.cpp:558]: this should be `anchors.fill: parent`

Comment (jmenghan) [reply source/qml/Library/RendererQml/AdaptiveCardQmlRenderer.cpp:558]: This happens when I used anchors.parent

image

The rectangle gets squished

File: source/qml/Library/RendererQml/Images/arrowdownIcon.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg width="16" height="16" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg"><title>navigation/arrow-down_16</title><path d="M8.5001 11.26336438l6.134-6.60428553c.188-.20253009.505-.21355895.707-.02506561.201.18849335.213.50632522.025.7088553l-6.5 6.997314214c-.095.1022676673-.227.159417246-.366.159417246-.139 0-.271-.0571495787-.366-.159417246L1.6311 5.34286854c-.09-.09725454-.134-.21957469-.134-.34189484 0-.13435165.053-.26870329.159-.36696046.202-.18849334.518-.17746448.706.02506561l6.138 6.60428553z" fill="#000" fill-rule="evenodd"/></svg>
```
Comment (rj100189) [inline source/qml/Library/RendererQml/Images/arrowdownIcon.svg:1]: Please pull in the latest code and I have included `arrow-down_12.svg`. Use that instead

Comment (jmenghan) [reply source/qml/Library/RendererQml/Images/arrowdownIcon.svg:1]: okay

File: source/qml/Library/RendererQml/AdaptiveCardQmlRenderer.cpp [hunk 3]
```diff
@@ -569,56 +562,71 @@ namespace RendererQml
...
 		uiComboBox->Property("valueRole", "'value'");
 		uiComboBox->Property("width", "parent.width");
 		//TODO : Add Height
-				
-		uiComboBox->Property("model", GetModel(choiceset.choices));
 
-        const auto textColor = choiceset.choices[0].textColor;
+		auto dropIcon = std::make_shared<QmlTag>("Image");
+		dropIcon->Property("source", Formatter() << "\"" << RendererQml::arrow_down_12 << "\"");
+		dropIcon->Property("anchors.right", "parent.right");
```
Comment (rj100189) [inline source/qml/Library/RendererQml/AdaptiveCardQmlRenderer.cpp:604]: is this needed?

File: source/qml/Library/RendererQml/Images/arrowdownIcon.svg (hunk 2 above)
Comment (rj100189) [reply source/qml/Library/RendererQml/Images/arrowdownIcon.svg:1]: Delete this as we don't need this anymore

Comment (jmenghan) [reply source/qml/Library/RendererQml/Images/arrowdownIcon.svg:1]: Sure

File: source/qml/Library/RendererQml/AdaptiveCardQmlRenderer.cpp (hunk 3 above)
Comment (jmenghan) [reply source/qml/Library/RendererQml/AdaptiveCardQmlRenderer.cpp:604]: yes to keep the icon in the right most corner

File: source/qml/Library/RendererQml/Images/arrowdownIcon.svg (hunk 2 above)
Comment (jmenghan) [reply source/qml/Library/RendererQml/Images/arrowdownIcon.svg]: done

**Review by rj100189 — APPROVED** [review]

**Pull request merged**